tests/scripts/common_functions.sh

```diff
@@ -88,13 +88,29 @@ function setup() {
 	esac
 }
 
+# Check if the prometheus is already deployed , if not invoke the script to deploy prometheus on minikube
+function setup_prometheus() {
+	kubectl_cmd="kubectl"
+	prometheus_pod_running=$(${kubectl_cmd} get pods --all-namespaces | grep "prometheus-k8s-1")
+	if [ "${prometheus_pod_running}" == "" ]; then
+		./scripts/prometheus_on_minikube.sh -as
```

Comment:
We will also need to add prometheus cleanup too. Did you debug the issue with terminating prometheus?